code/game/gamemodes/cult/runes.dm

```diff
@@ -945,6 +947,10 @@ var/list/teleport_runes = list()
 	notify_ghosts("Manifest rune created in [get_area(src)].", 'sound/effects/ghost2.ogg', source = src)
 
 /obj/effect/rune/manifest/can_invoke(mob/living/user)
+	if(is_busy)
+		to_chat(user,"<span class='cultitalic'>The rune is already in use.</span>")
```

Comment:
If you want to prevent this from letting people survive forever, you could replace `summoned_guy` with a list, and loop through that in rune deletion instead.